cmd/qat_plugin/dpdkdrv/dpdkdrv_test.go

```diff
@@ -63,10 +64,11 @@ func createTestFiles(prefix string, dirs []string, files map[string][]byte, syml
 
 func TestNewDevicePlugin(t *testing.T) {
 	tcases := []struct {
-		name            string
-		dpdkDriver      string
-		kernelVfDrivers string
-		expectedErr     bool
+		name                      string
+		dpdkDriver                string
+		kernelVfDrivers           string
+		preferredAllocationPolicy string
```

Comment:
without "Wrong `preferredAllocationPolicy`" test this parameter is not needed

Reply:
Let me add it.!
I know it is not needed if there is no test, but the 101st line (`_, err := NewDevicePlugin(1, tt.kernelVfDrivers, tt.dpdkDriver, tt.preferredAllocationPolicy)`) was complaining if there is no.

Reply:
Or, is the test unnecessary?